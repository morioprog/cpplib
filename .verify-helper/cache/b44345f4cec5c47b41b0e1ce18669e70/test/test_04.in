6
-4 5
1 2
-3 8
8 9
-8 1
0 6
