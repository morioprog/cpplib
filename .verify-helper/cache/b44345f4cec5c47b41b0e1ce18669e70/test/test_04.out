8 9
1 2
0 6
-3 8
-4 5
-8 1
